tests/thread_exit.c

```diff
@@ -0,0 +1,52 @@
+#include <stdc.h>
+#include <clock.h>
+#include <thread.h>
+#include <sched.h>
+#include <ktest.h>
+
+int exit_time[] = {100, 200, 150};
```

Comment:
All (?) global variables in test modules should be defined static to avoid symbol clashes.